pkg/device_plugin/device_plugin.go

```diff
@@ -315,6 +315,7 @@ func getDeviceName(deviceID string) string {
 				return deviceName
 			}
 			deviceName = strings.TrimSpace(splits[1])
+			deviceName = strings.ToUpper(deviceName)
 			reg, _ := regexp.Compile("\\s+")
 			deviceName = reg.ReplaceAllString(deviceName, "_") // Replace all spaces with underscore
```

Comment:
'.' and '/' are also replaced by '_'.  Might as well include that while we're fixing the upper case issue.

```go
	varName = strings.Replace(varName, "/", "_", -1)
	varName = strings.Replace(varName, ".", "_", -1)
```